src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -129,32 +129,55 @@ class HoldRequest extends React.Component {
     axios
       .get(`${appConfig.baseUrl}/api/newHold?itemId=${itemId}&pickupLocation=` +
         `${this.state.delivery}&itemSource=${itemSource}`)
-      .then(response => {
+      .then((response) => {
         if (response.data.error && response.data.error.status !== 200) {
           this.updateIsLoadingState(false);
           this.context.router.push(
             `${path}?errorStatus=${response.data.error.status}` +
             `&errorMessage=${response.data.error.statusText}${searchKeywordsQueryPhysical}` +
-            `${fromUrlQuery}`
+            `${fromUrlQuery}`,
           );
         } else {
           this.updateIsLoadingState(false);
           this.context.router.push(
             `${path}?pickupLocation=${response.data.pickupLocation}&requestId=${response.data.id}` +
-            `${searchKeywordsQueryPhysical}${fromUrlQuery}`
+            `${searchKeywordsQueryPhysical}${fromUrlQuery}`,
           );
         }
       })
-      .catch(error => {
+      .catch((error) => {
         console.error('Error attempting to make an ajax Hold Request in HoldRequest', error);
 
         this.updateIsLoadingState(false);
         this.context.router.push(
-          `${path}?errorMessage=${error}${searchKeywordsQueryPhysical}${fromUrlQuery}`
+          `${path}?errorMessage=${error}${searchKeywordsQueryPhysical}${fromUrlQuery}`,
         );
       });
   }
 
+  /**
+   * getNotification()
+   * Renders notification text surrounded by a 'nypl-banner-alert' toolkit wrapper.
+   *
+   * @return {HTML Element}
+   */
+  getNotification() {
+    return (
+      <div className="nypl-banner-alert">
+        <p style={{ padding: '10px 20px 0px', margin: 0 }}>
+          Please note that due to the holiday, items requested between Friday afternoon (1/12), and Tuesday afternoon (1/16),
+          will be delivered on Wednesday morning (1/17). Please check your patron account to be sure items are Ready for Pickup before your visit.
+        </p>
+        <p style={{ padding: '10px 20px', margin: 0 }}>
+          The Miriam and Ira D. Wallach Division Art and Architecture Reading Room (300) will be closed from January 8-14, 2018.
+          During this time, room 300 services will be available in room 308 on a limited basis.
+          Room 300 will reopen on Tuesday, January 16, 2018.
+          If you have any questions, please contact: <a href="mailto:art@nypl.org" target="_top">art@nypl.org</a>.
```

Comment:
Do the above text portions of the 'p' tags need to break earlier to conform to ESLint max line length?

Reply:
What do you mean? I'm not sure why the text was broken in those specific places but it's probably just for ease of read? It doesn't make a difference and eslint doesn't complain about the text content.